src/community/jms-cluster/jms-geoserver/src/main/java/org/geoserver/cluster/impl/handlers/catalog/CatalogUtils.java

```diff
@@ -6,14 +6,18 @@
 package org.geoserver.cluster.impl.handlers.catalog;
```

Comment:
Not sure what happen here, was the class reformat, is there any actual code change?